packages/path_provider/path_provider_windows/lib/folders.dart

```diff
@@ -0,0 +1,240 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:win32/win32.dart';
+
+/// A class containing the GUID references for each of the documented Windows
+/// known folders. A property of this class may be passed to the `getPath`
+/// method in the [PathProvidersWindows] class to retrieve a known folder from
+/// Windows.
+class WindowsKnownFolder {
```

Comment:
Is using a class as a namespace a common practice in Dart? I would have expected these to just be file-level constants.

Reply:
https://dart.dev/guides/language/effective-dart/design#avoid-defining-a-class-that-contains-only-static-members

Reply:
Ah, I was thinking of this as matching the clause, "However, this isn’t a hard rule. With constants and enum-like types, it may be natural to group them in a class", since this is the moral equivalent of an enum. But I can change it if we don't think this is a good fit.

Reply:
It can be nicer for autocomplete too, especially if they are related